Facebook.Unity/Assets/Facebook/Editor/UWPFacebookPostprocess.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEngine;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using System.Collections;
+using System.IO;
+
+public static class UWPFacebookPostprocess
+{
+    private const string FacebookUWPPath = "Assets/UWPFacebook/";
+
+    [PostProcessBuild(100)]
+    public static void OnPostProcessBuild(BuildTarget target, string path)
+    {
+        //We copy all files in the directory "../Plugins/UWP" because the winsdkfb requires the .windmd, .dll and .pri file along with the files inside the "Facebook" folder
```

Comment:
This will get run on all platforms. We should restrict this to only run on Windows.